src/utils/v2/veNft.ts

```diff
@@ -0,0 +1,9 @@
+import { VeNftVariant } from 'models/v2/veNftVariant'
+
+export const getNFTBaseImage = (
+  baseImagesHash: string,
+  variant: VeNftVariant,
+): string => {
+  const padded = variant.id.toString().padStart(2, '0')
+  return `https://${process.env.NEXT_PUBLIC_PINATA_GATEWAY_HOSTNAME}/ipfs/${baseImagesHash}/${padded}.png`
```

Comment:
**question**: this endpoint is heavily rate-limited. I wonder what else we might do here... 🤔

Reply:
my idea was to make a single large tiled image, similar to a sprite sheet, and have the frontend load and parse it based on known dimensions.

this was left out of MVP as an optimization step, but we can create an issue to track it.

Reply:
Partially tracked in #1526, launching and retrieving is part of #1416 which is an epic-level issue that i need to scope and will require discussions with JB contract folks.